rpm/calligra.spec

```diff
@@ -237,6 +213,24 @@ if [ -d %{buildroot}%{_libdir}/plugins/calligrastage ] ; then mv %{buildroot}%{_
 %post -p /sbin/ldconfig
 %postun -p /sbin/ldconfig
 
+%post filters -p /sbin/ldconfig
+%postun filters -p /sbin/ldconfig
+
+%post libs -p /sbin/ldconfig
+%postun libs -p /sbin/ldconfig
+
+%post plugins -p /sbin/ldconfig
+%postun plugins -p /sbin/ldconfig
```

Comment:
Actually this doesn't seem to have anything in /usr/lib(64) so no need to run ldconfig.